### PR TITLE
Docs: Add containerd setup instructions for Ubuntu 18.04/20.04

### PR DESCRIPTION
**Pull request by karras**
This PR adds new containerd setup instructions for Ubuntu 18.04 and 20.04. It proposes to switch to the containerd package provided by Ubuntu.

### CODE REVIEW

File: content/en/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -124,6 +124,25 @@ sudo mkdir -p /etc/containerd
...
+```shell
+# Restart containerd
+sudo systemctl restart containerd
+```
+{{% /tab %}}
+{{% tab name="Ubuntu 18.04/20.04" %}}
+
+```shell
+# (Install containerd)
+## Install containerd
```
Comment (kbhawkey) [inline content/en/docs/setup/production-environment/container-runtimes.md:136]: 👋  @karras 
I suppose you could remove the second comment, "Install containerd"
Question: Are the Docker steps required for 18.04/20.04? I have not tried to run these commands.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-24776--kubernetes-io-master-staging.netlify.app/docs/setup/production-environment/container-runtimes/#containerd

File: content/en/docs/setup/production-environment/container-runtimes.md (hunk 1 above)
Comment (karras) [reply content/en/docs/setup/production-environment/container-runtimes.md]: Done, I've removed the second "Install containerd" comment. The Docker related steps are not required anymore as the containerd package from the official Ubuntu repositories is quite recent.

Comment (kbhawkey) [reply content/en/docs/setup/production-environment/container-runtimes.md]: OK

**Review by irvifa — COMMENTED** [review]
/approve

**Pull request merged**